.github/workflows/ci.yml

```diff
@@ -12,33 +12,24 @@ jobs:
     name: Main Process
     runs-on: ubuntu-latest
     env:
-      GO_VERSION: '1.20'
-      GOLANGCI_LINT_VERSION: v1.52.2
+      GO_VERSION: '1.22'
```

Comment:
```suggestion
      GO_VERSION: stable
```

Reply:
@ldez:  stable does not work, as you can see [here](https://github.com/traefik/mocktail/actions/runs/7962385060/job/21735736711?pr=15).

Reply:
ah, I found a way to make it work :)